README.md

```diff
@@ -184,6 +184,7 @@ finished with errors
 * `--record` (`-r`): ask Kombucha to rewrite all the stored snapshots it knows about. Useful for setting a new baseline for tests or if you know a large amount of the API under test has changed
 * `--snapshots-directory` (`-s`): tell Kombucha where to find snapshots for the test run. If not specified, the default is `./__Snapshots/`.
 * `--work-directory` (`-w`): tell Kombucha what directory to use as the “work directory” (storage for live responses) for the test run. If not specified, the default is `./__Work__/`.
+* `--junit-output-url` (`-j`): If a path to a file (existing or not) is specified, this will produce a JUnitXML summary of the test run.
```

Comment:
I'm not so much a fan of this interface – while descriptive, it will restrict any future summaries/reporting styles we may want to support in the future. I'd prefer something like `--report [human|junit]` and/or `--output PATH` much like how xcpretty does this – it'll give us more options down the road and a nicer interface without over complicating things for users.

Reply:
Take a look :)